### PR TITLE
Off season2020-21 Commit

### PR DESCRIPTION
**Pull request by LakshBhambhani**
This commit looks big because old projects were deleted and converted to 2020 version. Furthermore, MustangLib was completed by bringing all features that we do not need access to in the main project

### CODE REVIEW

File: MustangLib-Imported/.vscode/settings.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
     "**/CVS": true,
     "**/.DS_Store": true,
```
Comment (adias408) [inline MustangLib-Imported/.vscode/settings.json:8]: what is MustangLib-Imported supposed to represent as opposed to MustangLib?

File: MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangCommand.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+
+/**
+ * Represents a robot action with defined health requirements for every subsystem it uses.
+ * 
+ * @author ctychen, lakshbhambhani
+ */
+public interface MustangCommand{
```
Comment (adias408) [inline MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangCommand.java:16]: we already had a MustangCommand, right? I'm wondering why this is a new file and wasn't the old file being moved

File: MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangScheduler.java [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+import java.util.Map;
+
+/**
+ * Responsible for scheduling and running commands, including
+ * MustangCommandBases. Use this instead of the WPILib CommandScheduler.
+ * 
+ * @author ctychen, lakshbhambhani
+ */
+
+public class MustangScheduler {
```
Comment (adias408) [inline MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangScheduler.java:23]: same question here - why is this a new file as opposed to a rename/move of the old file?

**Review by adias408 — COMMENTED** [review]
My main comment is that many of these files are added as if they're new but they're not - they're just moved from a different location. Doing it this way loses the history. Is there a reason for this?

**Comment by LakshBhambhani** [comment]
'MustangLib-Imported' is the name given by the WPiLib tool when it converts an old project to "2020 format". That is also the reason why MustangCommand and MustangScheduler are 2 new files rather than rename/move

**Comment by adias408** [comment]
> 'MustangLib-Imported' is the name given by the WPiLib tool when it converts an old project to "2020 format". That is also the reason why MustangCommand and MustangScheduler are 2 new files rather than rename/move

And you can't control the name? What does that tool do in converting? Why can't you simply rename it? Even if we somehow need to stick with the name "*-Imported" (which I tend to doubt), you're still deciding whether to do a "git add" or a "git mv"

**Comment by LakshBhambhani** [comment]
The name can definitely be changed and I will do that. That tool makes a new project and puts all of the files from the old project into the new one as new files.

**Pull request closed without merging**